docs/user_guide.md

```diff
@@ -4,63 +4,146 @@ This is a pre-release version of the extension and is not intended for general u
+
+### Security labels for columns
+
+Each personal table has one or more _protected entities_ associated with it. A protected entity is normally a person or a something associated with a person (a device or an account), but it could also be other things such as a household, a store, or a company.
 
-Regular users can be marked with the anoymization labels `direct`, `publish_trusted` or `publish_untrusted`.
-The value of the custom variable `pg_diffix.default_access_level` determines the access level for unlabeled regular users.
+Each protected entity must have at least one column that contains the identifier of the protected entity. We refer to these columns generally as AID columns (Anonymization ID). See (How to select AID columns)[#how-to-select-aid-columns] for more guidance.
+
+NOTE: if AID columns are not correctly labeled, the extension may fail to anonymize appropriately.
+
+The procedure `diffix.mark_personal(namespace, table_name, aid_columns...)` is used to label a table as personal and to label its AID columns.
+
+For example, 
 
 ```SQL
-SECURITY LABEL FOR pg_diffix ON ROLE analyst IS 'publish_trusted';
+CALL diffix.mark_personal('public', 'employee_info', 'employee_id');
 ```
 
-### Restricted features and extensions
+labels the table `employee_info` as personal, and labels the `employee_id` column as `employee_id` an AID column.
+
+```SQL
+CALL diffix.mark_personal('public', 'transactions', 'sender_acct', 'receiver_acct');
+```
+
+labels the table `transactions` as personal, and labels the `sender_acct` and `receiver_acct` columns as AID columns.
+
+## Settings
+
+The extension exposes a number of custom variables under the `pg_diffix` prefix.
+Superusers can change these variables at runtime for their own session,
+while regular users only have read access to them (with few notable exceptions).
+To use different values for all future sessions, they have to be set in the configuration file.
+
+Execute `SELECT * FROM diffix.show_settings();` to display the current settings of the extension.
+If the result is empty, make sure [`pg_diffix` is loaded](#using-the-extension).
+
+### Default behavior settings
+
+`pg_diffix.default_access_level` - Determines the default security label for unlabeled users. Default value is `direct`.
+
+`pg_diffix.session_access_level` - The security label that is active for the user's current session. This is initially set to the security label associated with the user, but the user can modify it to a value that has lower privilege than the user's normal security label. Privilege levels from high to low are `direct`, `anonymized_trusted`, `anonymized_untrusted`.
+
+`pg_diffix.treat_unmarked_tables_as_public` - Determines the default security label for tables. If set to `true`, then the default security label is `public`.  The default value is `false`.
```

Comment:
> Determines the default security label for tables.

A table being personal or public is not a security label. We could call this "protection" or "visibility". I'm not sure what a good name for this is.